ShipperOptions.json

```diff
@@ -1964,6 +1964,26 @@
         "value": "FEDEX_PTP_GROUND",
         "display": "FedEx® Ground®"
       },
+      {
+        "value": "FEDEX_PTP_PRIORITY_OVERNIGHT_ONE_RATE",
+        "display": "FedEx® Priority Overnight® One Rate®"
+      },
+      {
+        "value": "FEDEX_PTP_STANDARD_OVERNIGHT_ONE_RATE",
+        "display": "FedEx® Standard Overnight® One Rate®"
+      },
+      {
+        "value": "FEDEX_PTP_SECOND_DAY_AM_ONE_RATE",
+        "display": "FedEx® 2Day®A.M. One Rate®"
+      },
+      {
+        "value": "FEDEX_PTP_SECOND_DAY_ONE_RATE",
+        "display": "FedEx® 2Day® One Rate®"
+      },
+      {
+        "value": "FEDEX_PTP_EXPRESS_SAVER_ONE_RATE",
+        "display": "FedEx® Express Saver®"
```

Comment:
Should the display value also say `One Rate`?

Reply:
also, blame the beer.